TaniaGayosso/HackerRank_August25/IfElse.java

```diff
@@ -0,0 +1,23 @@
+package challenges.one;
+
+import java.util.Scanner;
+
+public class IfElse {
+
+	private static final Scanner scanner = new Scanner(System.in);
+	public static final String WEIRD_NUMBER = "Weird";
+	public static final String NOT_WEIRD_NUMBER = "Not Weird"; 
+	
+	public static void main(String[] args) {
+
+		int number = scanner.nextInt();
+		scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");
+
+		if (number % 2 == 1 || number >= 6 && number <= 20)  {
```

Comment:
If  n is even and in the inclusive range of  to , print Weird

What if number == 6